### ISSUE TITLE
Keyboard focusable scrollers

### ISSUE THREAD

**Issue by EllyLoel** [web-platform-tests/interop#1089]
### Description

This is a reprise of [this request made for Interop2025](https://github.com/web-platform-tests/interop/issues/762).

Scroll containers need to be focusable by default.

> I have spent a few years banging on about ensuring scrolling areas on a page are accessible to keyboard-only users. This is partly because the term [“keyboard” maps to other input types](https://www.w3.org/WAI/WCAG21/Understanding/keyboard.html#intent) that we distill to “keyboard” for ease of reference (speech input, sip-and-puff, on-screen keyboards, scanning software, etc.). When something cannot be operated with the keyboard, a whole bunch of users are blocked.

— Adrian Roselli, [Keyboard-Only Scrolling Areas](https://adrianroselli.com/2022/06/keyboard-only-scrolling-areas.html)

Scroll containers have been focusable in Firefox since version 4 and Chrome since version 132.

https://developer.chrome.com/blog/keyboard-focusable-scrollers

Safari does not make scroll containers focusable at all, regardless of whether the "Press Tab to highlight each item on a web page" Safari setting and the Full Keyboard Access macOS/iOS/iPadOS settings are turned on.

### Specification

https://html.spec.whatwg.org/#focusable-area

### Additional Signals

<dl>
  <dt>Chrome Platform Status:<dt>
  <dd><a href="https://chromestatus.com/feature/5231964663578624">https://chromestatus.com/feature/5231964663578624</a></dd>
  <dt>Webkit Bugzilla:</dt>
  <dd><a href="https://bugs.webkit.org/show_bug.cgi?id=190870">https://bugs.webkit.org/show_bug.cgi?id=190870</a></dd>
  <dd><a href="https://bugs.webkit.org/show_bug.cgi?id=277290">https://bugs.webkit.org/show_bug.cgi?id=277290</a></dd>
</dl>

**Comment by yatil** [comment]
Explicitly expressing support for this proposal.

**Comment by jgraham** [comment]
Is this something that is tested in web-platform-tests or that you can write a web-platform-test for? Otherwise it's hard to consider the proposal, just because we can't measure whether the change was made or not.

I think in principle the way to write such a test would involve sending tab keystrokes via testdriver (i.e. webdriver) and checking which elements get focus, but I'm not 100% sure if that works in practice in all implementations.

**Comment by mfreed7** [comment]
FYI, I put up a CL to move Chromium's (currently-internal) tests for keyboard focusable scrollers out to WPT:

https://chromium-review.googlesource.com/c/chromium/src/+/7275628

**Comment by wpt-interop** [comment]
Thank you for proposing Keyboard focusable scrollers for Interop 2026.

We want to let you know it was not selected to be part of Interop this year.

On behalf of the entire Interop team, thank you for submitting it for consideration. We got many more proposals than we could include in this year's project, necessitating some difficult choices.  Please note this should not be taken as a comment on the technology as a whole, or our willingness to consider it for Interop in the future.  We appreciate the work you put into your proposal, and would welcome your participation in future rounds of Interop.

_Posted on behalf of the Interop team._
